src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/endpoint/ExternalDocRequestEndpointTest.kt

```diff
@@ -69,10 +70,26 @@ internal class ExternalDocRequestEndpointTest {
         verifyNoMoreInteractions(sqsService, telemetryService)
     }
 
+    @Test
+    fun `given a valid message with dummy court room then should enqueue the message and return the correct acknowledgement`() {
```

Comment:
Should this not be
```suggestion
    fun `given a valid message with dummy court room then should *not* enqueue the message and return the correct acknowledgement`() {
```
?

Reply:
Correct - sorry